web-test-runner.config.mjs

```diff
@@ -0,0 +1,17 @@
+import {importMapsPlugin} from '@web/dev-server-import-maps';
+
+export default {
+	nodeResolve: true,
+	plugins: [
+		importMapsPlugin({
+			inject: {
+				importMap: {
+					imports: {
+						'leaflet': '/src/Leaflet.js',
+						'chai': '/node_modules/chai/chai.js'
```

Comment:
@mourner this seems to have fixed the issue, not sure why it didn't work out of the box here.